src/main/java/cn/nukkit/item/Item.java

```diff
@@ -399,7 +399,7 @@ private static void initCreativeItems() {
         // Creative inventory for 407+
         for (Map map : new Config(Config.YAML).loadFromStream(Server.class.getClassLoader().getResourceAsStream("creativeitems407.json")).getMapList("items")) {
             try {
-                addCreativeItem(v1_16_0, Item.get((int) map.get("id"), (int) map.getOrDefault("damage", 0), 1, map.get("nbt_hex") != null ? Utils.parseHexBinary((String) map.get("nbt_hex")) : new byte[0]));
+                addCreativeItem(v1_16_0, fromJson(map));
```

Comment:
In creativeitems407.json, some items use nbt_b64 to save nbt data

Reply:
The previous version can use fromJsonOld but I think fromJson is compatible with old data (Google Translate)